guides/common/modules/proc_using-vmware-cloud-init-and-userdata-templates-for-provisioning.adoc

```diff
@@ -30,7 +30,7 @@ When you set up the compute resource and images for VMware provisioning in {Proj
 * Configure port and firewall settings to open any necessary connections.
 Because of the `cloud-init` service, the virtual machine always calls back to {Project} even if you register the virtual machine to {SmartProxy}.
 ifndef::orcharhino[]
-For more information about port and firewall requirements, see {InstallingServerDocURL}Ports_and_Firewalls_Requirements_{project-context}[Port and Firewall Requirements] in _{InstallingServerDocTitle}_ and {InstallingSmartProxyDocURL}capsule-ports-and-firewalls-requirements_{smart-proxy-context}[Ports and Firewalls Requirements] in _{InstallingSmartProxyDocTitle}_.
+For more information about port and firewall requirements, see {InstallingServerDocURL}Port_and_firewalls_requirements_{project-context}[Port and firewall requirements] in _{InstallingServerDocTitle}_ and {InstallingSmartProxyDocURL}capsule-ports-and-firewalls-requirements_{smart-proxy-context}[Ports and firewalls requirements] in _{InstallingSmartProxyDocTitle}_.
```

Comment:
Same comment as above. Fix URL and link text - "port", "firewall"

Reply:
Fixed.